### PR TITLE
[zh]translate kubernetes-api/workload-resources/priority-class-v1.md into chinese

### PR DESCRIPTION
**Pull request by twilight0620**
Fixes: https://github.com/kubernetes/website/issues/33646

Do not have chinese docs:  https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/priority-class-v1/

so need to translatekubernetes-api/workload-resources/priority-class-v1.md into Chinese

### CODE REVIEW

File: content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md [hunk 1]
```diff
@@ -0,0 +1,832 @@
...
+
+  标准对象的元数据。
+  更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+
+<!--
+- **value** (int32), required
+
+  The value of this priority class. This is the actual priority that pods receive when they have the name of this class in their pod spec.
+-->
+- **value** (int32)，必填
```
Comment (howieyuen) [inline content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md:64]: same as below
```suggestion
- **value** (int32)，必需
```

File: content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md [hunk 2]
```diff
@@ -0,0 +1,832 @@
...
+
+  此优先级的值。这是 pod 在其 pod 规范中有此类名称时收到的实际优先级。
```
Comment (howieyuen) [inline content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md:66]: keep initials capitalized of k8s resource kind, same as below
```suggestion
  此优先级的值。这是 Pod 在其 Pod 规范中有此类名称时收到的实际优先级。
```

File: content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md [hunk 3]
```diff
@@ -0,0 +1,832 @@
...
+PriorityClass defines mapping from a priority class name to the priority integer value. The value can be any valid integer.
+
+<hr>
+ -->
+PriorityClass 定义了从优先级类名称到优先级整数值的映射。
+该值可以是任何有效的整数。
+
+<hr>
+
+<!-- - **apiVersion**: scheduling.k8s.io/v1 -->
```
Comment (howieyuen) [inline content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md:43]: 译文与原文一致，可以删除原文对照，下同

File: content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md [hunk 4]
```diff
@@ -0,0 +1,832 @@
...
+  
+<!--
+- **preemptionPolicy** (string)
+
+  PreemptionPolicy is the Policy for preempting pods with lower priority. One of Never, PreemptLowerPriority. Defaults to PreemptLowerPriority if unset.
+-->  
+- **preemptionPolicy** (string)
+
+  PreemptionPolicy 是抢占优先级较低的 Pod 的策略。
+  Never、PreemptLowerPriority 之一。
```
Comment (howieyuen) [inline content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md:97]:
```suggestion
  可选值：Never、PreemptLowerPriority。
```

File: content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md [hunk 5]
```diff
@@ -0,0 +1,832 @@
...
+  如果未设置，则默认为 PreemptLowerPriority。
+
+
```
Comment (howieyuen) [inline content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md:100]: remove extra empty lines, same as below

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md (hunk 2 above)
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md:66]: spec 统一译为规约

File: content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md [hunk 6]
```diff
@@ -0,0 +1,832 @@
...
+
+<hr>
+
+
+
+
+
+<!-- ### `get` read the specified PriorityClass -->
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md:159]: 这类 `<!--` 和 `-->` 都单独占一行，以便将来变更的时候逐行比较。下同。

File: content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md [hunk 7]
```diff
@@ -0,0 +1,832 @@
...
+
+
+200 (<a href="{{< ref "../workload-resources/priority-class-v1#PriorityClass" >}}">PriorityClass</a>): OK
+
+401: Unauthorized
+-->
+#### 响应
+
+
+200 (<a href="{{< ref "../workload-resources/priority-class-v1#PriorityClass" >}}">PriorityClass</a>): 成功
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md:204]: OK 不要翻译

File: content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md [hunk 8]
```diff
@@ -0,0 +1,832 @@
...
+
+401: 未授权
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md:206]: 这种 HTTP 响应含义不要翻译。

```suggestion
401: Unauthorized
```

File: content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md [hunk 9]
```diff
@@ -0,0 +1,832 @@
...
+#### 响应
+
+
+200 (<a href="{{< ref "../workload-resources/priority-class-v1#PriorityClass" >}}">PriorityClass</a>): 成功
+
+201 (<a href="{{< ref "../workload-resources/priority-class-v1#PriorityClass" >}}">PriorityClass</a>): 已创建
+
+202 (<a href="{{< ref "../workload-resources/priority-class-v1#PriorityClass" >}}">PriorityClass</a>): 已接受
+
+401: 未授权
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md:410]: 同上，冒号后面的单词不要翻译

File: content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md (hunk 1 above)
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md:64]: 必填也行吧？我其他翻译的，comments让我写的必填。

**Comment by twilight0620** [comment]
I have modified the comments, thanks for reviewing.

File: content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md:70]: 还是统一为 “必需” 好些

File: content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md [hunk 10]
```diff
@@ -0,0 +1,864 @@
...
+
+  items is the list of PriorityClasses
+-->
+- **items** ([]<a href="{{< ref "../workload-resources/priority-class-v1#PriorityClass" >}}">PriorityClass</a>)，必填
+
+  items 是 PriorityClasses 的列表
+
+
+
+
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/workload-resources/priority-class-v1.md:150]: 类似这种空行，保留一个就好，剩下的删掉。下同。

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**